test/utils/structs/DoubleEndedQueue.test.js

```diff
@@ -11,7 +10,7 @@ async function getContent (deque) {
 }
 
 /** Revert handler that supports custom errors. */
-async function expectRevert (promise, reason) {
+expectRevert.customError = async function (promise, reason) {
```

Comment:
We are overwriting this property in both places. It should only be defined in one place, it's a shared global object.